com.aobuchow.sample.commander/src/com/aobuchow/sample/commander/FileClipBoard.java

```diff
@@ -42,6 +42,9 @@ public void cut(IResource[] resourcesToCut) {
 	}
 
 	public IResource paste(IContainer destination, Shell shell) {
+		if (clipboard.isDisposed()) {
```

Comment:
This is trying to help prevent https://github.com/AObuchow/Sample-Commander/issues/31, although an error should probably be logged if this occurs as it shouldn't be occuring in the first place...